homeassistant/components/solaredge/const.py

```diff
@@ -150,35 +150,45 @@
         json_key="Purchased",
         name="Imported Power",
```

Comment:
Should we rename this? As Power is obviously not correct considering the unit of measurement.